.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: PR_build
```

Comment:
Is this temp or perm?

Reply:
permanent, its so we don't need to push to main everytime we want the workflow to build